code-challenges.js

```diff
@@ -31,18 +68,34 @@ var myNumbers1 = [3, 7, 0, 6, -9]
 
 var myNumbers2 = [8, -7, 0, 6, 2, 9, 13]
 
-
-
-
+//input numbers 
+//output numbers 
+// this is where i make my constant declaration 
+const oddNumbers = (numbers) => {
+    let num =[]
+     //this is where my loop will be to make the numbers odd using module to get the odds numbers
+    for(let i=0; i<numbers.lenght; i++){
+        if(number[i] % 2 ! =0){
+            num.push(array[i])
+        }
+    }
+    return num
+}
+console.log(oddNum(myNumbers2))
+
+//kept getting syntax error for the bang character couldn't figure out how to make it work
```

Comment:
Looks good. There is a space next to the bang operator where there should be another = sign. 
`number[i] % 2 !== 0`